libraries/adaptive-expressions/src/expressionProperties/expressionProperty.ts

```diff
@@ -13,38 +13,56 @@ import { Expression } from '../expression';
  */
 export class ExpressionProperty<T> {
     private defaultValue: T;
+    private expression: Expression;
 
     public constructor(value?: T | string | Expression, defaultValue?: T) {
         this.defaultValue = defaultValue;
         this.setValue(value);
     }
 
+    /**
+     * Gets or sets the raw value of the expression property.
+     */
     public value: T;
 
-    public expression: Expression;
+    /**
+     * Getes or sets the expression text to evaluate to get the value.
+     */
+    public expressionText: string;
 
     public toString(): string {
-        return this.expression ? `=${ this.expression.toString() }` : (this.value ? this.value.toString() : '');
+        if (this.expressionText) {
```

Comment:
Documentation.  All public classes / methods should have documentation.

Reply:
added doc.